lib/magma/attributes/link.rb

```diff
@@ -35,5 +35,9 @@ def link_records(identifiers)
     def link_identity
       link_model.attributes[link_model.identity]
     end
+
+    def missing_column?
+      false
+    end
```

Comment:
Since link attributes like `Magma::ChildAttribute` don't have columns on their model's tables, they'll always return false for `#missing_column?`.

Reply:
Magma::ForeignKeyAttributes are also link_attributes, meaning `parent` and `link` types require a column.

Reply:
Gotcha. This should be fixed in https://github.com/mountetna/magma/pull/152/commits/353ab82de2895033c0f2d88f08c4934bf4bd9b93.